corrcal/wrapper.py

```diff
@@ -18,7 +18,7 @@
 
 # With ctypes, the location of libcorrcal*.so need to be hard-coded.
 # This will not be needed once we migrate to CFFI.
-LIB_LOC = glob(site.getsitepackages()[0] + "/c_corrcal*.so")[0]
+LIB_LOC = glob(site.getsitepackages()[0] + "/corrcal/c_corrcal*.so")[0]
```

Comment:
I am getting issues with this import here.

Could you print the string stored under LIB_LOC.
Mine returns and empty list [].

Reply:
Hmm. It  is working fine on my machine. `LIB_LOC` can be access from `corrcal.wrapper.LIB_LOC`, and mine is pointing to the right file.

```python
In [10]: ls /Users/piyanat/src/miniconda3/envs/corrcal/lib/python3.7/site-packages/corrcal/                                                                                                          
__init__.py                      c_corrcal.cpython-37m-darwin.so* data/                            wrapper.py
__pycache__/                     corrcal.py                       test/

In [11]: import site                                                                                                                                                                                 

In [12]: site.getsitepackages()                                                                                                                                                                      
Out[12]: ['/Users/piyanat/src/miniconda3/envs/corrcal/lib/python3.7/site-packages']

In [13]: wrapper.LIB_LOC                                                                                                                                                                             
Out[13]: '/Users/piyanat/src/miniconda3/envs/corrcal/lib/python3.7/site-packages/corrcal/c_corrcal.cpython-37m-darwin.so'

```
 Can you:
- make sure that you have to most recent commit on restructure branch
- `pip uninstall corrcal` 
- reinstall it with `pip install .` from your local repository.
- launch `ipython` __*outside*__ of your local repository and try to import again.

Reply:
Did it and problem is resolved, thanks!

Reply:
The problem of it loading the local package rather than the installed one is a common problem and the most suggested fix is to put your actual package in a (non-python-package) `src/` folder. So you'd have:

```
corrcal/
    setup.py
    src/
        corrcal/
            __init__.py
```

This will also help in testing (because pytest will by default pick up the local package rather than the installed one, which is _usually_ not what you want!)

Reply:
Nice! Thanks for the suggestion @steven-murray